### ISSUE TITLE
More methods of earning Coral

### ISSUE THREAD

**Issue by rjtngit** [wetfish/online#58]
- Browsing - every page load has a % chance of earning coral, with a maximum amount allowed per day
- Thread replies - collect coral whenever someone replies to your thread
- Random chests - clickable treasure chest appears somewhere on the page
- Registration
- Daily login

**Comment by ghost** [comment]
Daily Login Bonus

@rempelj not sure if you have played mobile F2P games but they have a function where you log in every day and you are given a bonus for each reoccurring day that you login.

This one might be out there - Cooldown Bonus or some form of that?

If you are offline for X amount of hours then you are given a bonus for that time. I've seen this in some games as an incentive to get the fuck off occasionally so that you are given a higher return when you do come back to play.

**Comment by rjtngit** [comment]
Good idea! I think we should give MORE currency for consecutive login days. No leaving allowed!

**Comment by ghost** [comment]
Bonus would include coral 100, 200, 300, etc. I've seen things like a 3 x 3 sheet that are stamped on the day of. You can riddle it with free items and such. Once the 3 x 3 is completed then you begin getting significantly higher bonuses as it moves through the 3 x 3 again with 1000/2000 coral per day with very helpful items riddled.

**Comment by ghost** [comment]
Community Voting

**Comment by ghost** [comment]
bonus coral for being quoting? kitlero's idea im not certain if it's positive

**Comment by Kitlero** [comment]
I mean a tiny amount, like one coral per post if you're quoted. That way you don't get a bajillion quote chain long post to hack the coral system. 

Eg. Me > Hauyne > Rempelj > Me Would still only give one. 

Same for people who would quote someone six different times for the same post to pick apart an argument or something. Just one coral.

Might improve the chances of getting real substance responses since it would entice people to be quoted.

**Comment by ghost** [comment]
thank you for clarifying

**Comment by Kitlero** [comment]
Underneath my shit posting exterior I DO wish the best for Wetfish. O7
